ted/src/main/java/frc/robot/control/AutonomousChooser.java

```diff
@@ -104,8 +117,10 @@ public Command getAutoPath() {
      * @return command
      */
     public Command getCommand(){
+        subsystems.getCameraSubsystem().setBotPoseSource();
```

Comment:
add a comment about why this command needs to be called here (e.g. why it can't be called in robot container)

Reply:
added